src/index.ts

```diff
@@ -13,18 +13,17 @@ import {
 } from './types';
 
 type LoaderResult = Config | null;
+export type LoaderSync = (filepath: string, content: string) => LoaderResult;
```

Comment:
This reordering seems unnecessary, please revert to the original order

Reply:
Done. I originally did that because it broke `no-use-before-define` rule, but since you have it disabled for `@typescript/no-use-before-defined`, and disabled that one too.